.travis.yml

```diff
@@ -23,23 +23,84 @@ matrix:
   fast_finish: true
   include:
     -
-      bundler_args: 
+      bundler_args:
       dist: trusty
-      env: PUPPET_INSTALL_TYPE=agent BEAKER_debug=true BEAKER_PUPPET_COLLECTION=puppet6 BEAKER_set=docker/centos-7 BEAKER_TESTMODE=apply
-      rvm: 2.5.3
-      script: bundle exec rake beaker
+      env: PLATFORMS=deb_puppet5
+      rvm: 2.5.1
+      before_script:
+      - bundle exec rake 'litmus:provision_list[waffle_deb]'
+      - bundle exec rake 'litmus:install_agent[puppet5]'
+      - bundle exec rake litmus:install_module
+      script:
+      - bundle exec rake litmus:acceptance:parallel
       services: docker
-      stage: acceptance
       sudo: required
     -
-      bundler_args: 
+      bundler_args:
       dist: trusty
-      env: PUPPET_INSTALL_TYPE=agent BEAKER_debug=true BEAKER_PUPPET_COLLECTION=puppet6 BEAKER_set=docker/ubuntu-14.04 BEAKER_TESTMODE=apply
-      rvm: 2.5.3
-      script: bundle exec rake beaker
+      env: PLATFORMS=deb_puppet6
+      rvm: 2.5.1
```

Comment:
Just a minor comment, everything else LGTM.
rvm version in this file needs updated to 2.5.3 in this file.